docs/spec/api.md

```diff
@@ -618,26 +618,6 @@ The "digest" parameter must be included with the PUT request. Please see the
 [_Completed Upload_](#completed-upload) section for details on the parameters
 and expected responses.
 
-Additionally, the upload can be completed with a single `POST` request to
```

Comment:
at this point, we'd like to get all the repos onto the same process - and this one is the only outlier, meaning its also the only one that won't get automated docs updates.

once we use the same process, changing it is :+1: - but first I'd like to get docs updates out regularly.